docs/build/index.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Build circuits with Qiskit
+title: Introduction
 description: Build circuits with Qiskit and Qiskit Runtime
 
 ---
 
-# Build circuits with Qiskit
+# Introduction
 
-In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits.
\ No newline at end of file
+In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits. This documentation covers building circuits and operators in Qiskit, visualizing circuits, and other circuit-building tools.
```

Comment:
Maybe have the new sentence as a new paragraph? Make the content more skimmable.

Reply:
Done